test/unit/spec/util/index.js

```diff
@@ -92,4 +94,40 @@ describe('util', () => {
       return promise;
     });
   });
+
+  describe.only('chromeScreenShare', () => {
+    const labels = ['web-contents-media-stream://1174:3', 'window:1561:0', 'screen:2077749241:0'];
```

Comment:
So far you are testing for only +ve cases for the function for chrome. Please add some -ve cases as well.

Reply:
Added invalidLabels to the mix.